config/README.md

```diff
@@ -0,0 +1,100 @@
+Config. To learn more about how you can use Firebase Remote Config in your app,
+see
+[Firebase Remote Config Introduction](https://firebase.google.com/docs/remote-config/).
+
+Getting started
+---------------
+
+1. [Add Firebase to your Web Project](https://firebase.google.com/docs/web/setup).
+2. [Create a Remote Config project for the quickstart sample](https://firebase.google.com/docs/remote-config/android#create_a_product_name_project_for_the_quickstart_sample),
+  defining the parameter values and parameter keys used by the sample.
+    `welcome_message`
+3. Add the config object to index.html. See TODO in index.html.
+3. Run the sample with the server of your choice. For example, for Python's
+   SimpleHttpServer run `python -m SimpleHTTPServer 8080` from the sample
+   directory.
+4. Change the parameter value in the Firebase Console (the value of
+  `welcome_message`).
+5. Tap **Fetch Remote Welcome** in the app to fetch new parameter values and see
+  the resulting change in the app.
+
+Best practices
+--------------
+This section provides some additional information about how the quickstart
+example sets in-app default parameter values and fetches values from the Remote
+Config service
+
+### In-app default parameter values ###
+
+In-app default values are set using a JSON object using the `setDefaults` method.
+Then, you can override only those values that you need to change from the
+Firebase console. This lets you use Remote Config for any default value that you
+might want to override in the future, without the need to set all of those
+values in the Firebase console.
+
+### Fetch values from the Remote Config service ###
+
+When an app calls `fetch`, locally stored parameter values are used unless the
+minimum fetch interval is reached. The minimal fetch interval is determined by:
+
+1. The parameter passed to `fetch(long minFetchInterval)`.
+2. The minimum fetch interval set in Remote Config settings.
+3. The default minimum fetch interval, 12 hours.
+
+Fetched values are immediately activated when retrieved using `fetchAndActivate`.
+In the quickstart sample app, you call `fetchAndActivate` from the UI by tapping
+**Fetch Remote Welcome**.
+
+To control when fetched values are activated and available to your app use `fetch`.
+The values are locally stored, but not immediately activated. To activate fetched
+values so that they take effect, call the `activate` method.
+
+You can also create a Remote Config Setting to enable "developer mode". By
+using a small (even 0) value for `minimumFetchIntervalInSeconds`, you can make many
+more requests per hour, so you can test your app with different Remote Config
+parameter values during development.
+
+- To learn more about fetching data from remote config, see the Remote Config
+  Frequently Asked Question (FAQ) on
+  [fetching and activating parameter values](https://firebase.google.com/support/faq#remote-config-values).
```

Comment:
I think we could update this to the "Config loading strategies" page and not mention the FAQ.  WDYT?

Reply:
Should I remove/update the current FAQ link and then add a link to the "Config loading strategies" page? Please be more specific.

Reply:
Done